src/m365/spo/commands/list/list-roleinheritance-reset.ts

```diff
@@ -0,0 +1,96 @@
+import { Logger } from '../../../../cli';
+import { CommandOption } from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import { validation } from '../../../../utils';
+import SpoCommand from '../../../base/SpoCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+  listId?: string;
+  listTitle?: string;
+}
+
+class SpoListRoleInHeritanceResetCommand extends SpoCommand {
+  public get name(): string {
+    return commands.LIST_ROLEINHERITANCE_RESET;
+  }
+
+  public get description(): string {
+    return 'Restore role inheritance on list or library.';
+  }
+
+  public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
+    if (this.verbose) {
+      logger.logToStderr(`reset role inheritance of list in site at ${args.options.webUrl}...`);
+    }
+
+    let requestUrl: string = '';
```

Comment:
We can avoid repetition on lines 36 and 39 by setting the first part of the URL, which is the same for both ways to retrieve the list, as the default value for the variable and then append the proper suffix.

Reply:
done